projects/core/src/cart/store/effects/wish-list.effect.ts

```diff
@@ -45,9 +54,76 @@ export class WishListEffects {
     })
   );
 
+  @Effect()
+  loadWishList$: Observable<
+    | CartActions.LoadWisthListSuccess
+    | CartActions.CreateWishList
+    | CartActions.LoadCartFail
+  > = this.actions$.pipe(
+    ofType(CartActions.LOAD_WISH_LIST),
+    map((action: CartActions.LoadWisthList) => action.payload),
+    concatMap(userId => {
+      return this.cartConnector.loadAll(userId).pipe(
+        switchMap(carts => {
+          if (carts) {
+            const wishList = carts.find(cart => cart.name === 'wishlist');
```

Comment:
The cart name is always "wishlist"? In "createWishList", `name` is a parameter which can be anything, right?